server/utils/logger.js

```diff
@@ -103,7 +103,13 @@ try {
 } catch (error) {
   // no op
 }
-/* eslint-enable global-require */
-/* eslint-enable import/no-unresolved */
+
+export const logResponseError = (error) => {
+  if (error.response) {
+    const {status, data} = error.response;
+    if (status === 500) Logger.error(error.response);
```

Comment:
any 5xx error we should log with error level

Reply:
Resolved in c8e7a80